app/admin/source_aides.rb

```diff
@@ -8,7 +8,9 @@
   config.filters = false
 
   index do
-    column(:categorie) { |a| status_tag a.categorie }
+    column(:categorie) do |a|
+      render 'components/tag', image_path: '', contenu: t(".categories.#{a.categorie}"), classes: ''
```

Comment:
selon moi, il s'agit d'une `partial` : 

```suggestion
      render partial: 'components/tag', locals: { image_path: '', contenu: t(".categories.#{a.categorie}"), classes: '' }
```

Je ferai aussi en sorte que les paramètres `image_path` et `classes` soient factulatif : 

```suggestion
      render partial: 'components/tag', locals: { contenu: t(".categories.#{a.categorie}") }
```

pour faire cela, je te montre dans un autre commentaire